### PR TITLE
License cleanup following ext/ refactor

### PR DESCRIPTION
**Pull request by michdolan**
This PR:

- Removes licenses for projects that are no longer embedded in the OCIO repo
- Adds a license (from OpenImageIO contributed files) that was missing. This one is a bit of a gray area to me, since it matches the argparse license block except for the NVIDIA copyright.
- Changes the listed Imageworks copyright to a more inclusive "Contributors to the OpenColorIO project".
- Adds missing license notices to files, as found in the latest license scan

The file currently contains licenses for source files that were pulled into OpenColorIO from other projects by OpenColorIO. Files that were directly contributed by authors are included in the blanket "Contributors to the OpenColorIO Project" copyright.

### CODE REVIEW

**Comment by michdolan** [comment]
Thanks for taking a look @swinslow !

@lgritz Can you double check my changing the Imageworks copyright to the more inclusive "contributors" copyright? Seem OK to you?

**Comment by michdolan** [comment]
@jmertic , does tests/data/LICENSE.md accomplish what was discussed in TSC meeting regarding blanket license coverage for unit test data?

**Comment by jmertic** [comment]
Yes - this works @michdolan. Thanks and I will have a new scan after you merge.

**Comment by doug-walker** [comment]
There are some test files that have copyrights within them that conflict with the overall license file you added to the enclosing directory.  These are all files that we added as part of the Autodesk work, so we will do a follow-up PR to remove those or change them to not conflict.

File: LICENSE.md [hunk 1]
```diff
@@ -1,35 +1,17 @@
...
+All code by Contributors to the OpenColorIO Project except:
 
-Pystring
-https://github.com/imageworks/pystring
-
-yaml-cpp
-https://github.com/jbeder/yaml-cpp
-
-PTex (Mutex), courtesy of Brent Burley and Disney
-http://ptex.us/
```
[the remark this answers is not in the document]
Comment (michdolan) [reply LICENSE.md]: The PTex Mutex code was removed in #632 , and I could not find anything else referencing this license or copyright notice outside the LICENSE file itself.

[the remark this answers is not in the document]
Comment (doug-walker) [reply LICENSE.md]: Oh right, I remember now we talked about it.  LGTM

**Comment by michdolan** [comment]
Thanks @doug-walker . Regarding the rest of the license notices in cpp/h files throughout the repo: I think we will want to replace those with the simplified statement too, but since it is not blocking our license scan, I can do a followup PR simplifying all the others.

**Comment by michdolan** [comment]
This PR could be merged tomorrow if there is no additional feedback.

**Pull request merged**